### ISSUE TITLE
Add renaming on project select screen

### ISSUE THREAD

**Issue by Dizeeee** [godotengine/godot#10740]
It would be nice to have a button on the project select screen to be able to rename projects.

**Comment by nabor** [comment]
I would like to take this to get started on contributing.
I was looking at the source and have a couple of strategies I would like to discuss to take the best approach

**Comment by nabor** [comment]
Just wanted to ask what you think about changing the folder name of the project also
not suported, default or through a checkbox?

Also how can I print something to stderr so I can check small things without running the debugger?

**Comment by akien-mga** [comment]
> changing the folder name of the project also not suported, default or through a checkbox?

Could you rephrase? I'm not sure I understand.

**Comment by nabor** [comment]
Yeah sorry that's terrible
The question actually was
Should we also rename the folder containing the project?

**Comment by akien-mga** [comment]
I think it's better not to change it, we don't want to mess with users' filesystem IMO. It should only change the name in project.godot.

**Comment by nabor** [comment]
I'm trying to get the project name but I can't
This is what i have
```cpp
String dir = _test_path();
if (dir == "") {
	error->set_text(TTR("Invalid project path (changed anything?)."));
	return;
}

if (ProjectSettings::get_singleton()->setup(dir.plus_file("/project.godot"), "")) {
	error->set_text(TTR("Couldn't get project.godot in project path."));
} else {
	const String name_key = "application/config/name";
	if (ProjectSettings::get_singleton()->has(name_key)) {
		print_line("Has name");
		String appname = ProjectSettings::get_singleton()->get(name_key);
		print_line(appname);
	}
}
```
And it has the property but the get() returns an empty string
